apps/condo/domains/organization/integrations/sbbol/sync/syncClientAccounts.js

```diff
@@ -0,0 +1,86 @@
+const dayjs = require('dayjs')
+
+const conf = require('@open-condo/config')
+const { getLogger } = require('@open-condo/keystone/logging')
+const { get, isEmpty } = require('lodash')
+
+const { initSbbolFintechApi } = require('@condo/domains/organization/integrations/sbbol/SbbolFintechApi')
+const { dvSenderFields } = require('@condo/domains/organization/integrations/sbbol/constants')
+const { BankAccount } = require('@condo/domains/banking/utils/serverSchema')
+const { RUSSIA_COUNTRY } = require('@condo/domains/common/constants/countries')
+const { ISO_CODES_FOR_SBBOL } = require('@condo/domains/common/constants/currencies')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+
+const logger = getLogger('sbbol/syncClientAccounts')
+const SBBOL_FINTECH_CONFIG = conf.SBBOL_FINTECH_CONFIG ? JSON.parse(conf.SBBOL_FINTECH_CONFIG) : {}
+
+const sync = async (client) => {
+    const { keystone: UserContext } = await getSchemaCtx('User')
+    const { keystone: BankAccountContext } = await getSchemaCtx('BankAccount')
+
+    const [organization] = await Organization.getAll(UserContext,
+        { tin: client.inn },
+        { sortBy: 'createdAt_DESC' },
+    )
+
+    for (let accountIndex = 0; accountIndex <= client.accounts.length - 1; accountIndex++) {
+        const currentAccount = client.accounts[accountIndex]
+        const bankAccountDetails = {
+            tin: organization.tin,
+            country: RUSSIA_COUNTRY,
+            number: currentAccount.number,
+            currencyCode: (ISO_CODES_FOR_SBBOL[currentAccount.currencyCode]),
+            routingNumber: currentAccount.bic,
+        }
+        const foundAccount = await BankAccount.getOne(BankAccountContext, {
+            ...bankAccountDetails,
+            organization: { id_in: organization.id },
+        })
+        if (isEmpty(foundAccount)) {
+            await BankAccount.create(
+                BankAccountContext,
+                {
+                    meta: {
+                        sbbol: {
+                            type: currentAccount.type,
+                            state: currentAccount.state,
+                            openDate: currentAccount.openDate,
+                            name: currentAccount.name,
+                            closeDate: currentAccount.closeDate,
+                        },
+                    },
+                    ...bankAccountDetails,
+                    ...dvSenderFields,
+                    organization: { connect: { id: organization.id } },
+                }
+            )
+        }
+    }
+}
+/**
+ * Fetches ClientAccounts from SBBOL for specified date and creates BankAccount if this bank account has not yet been created
+ *
+ * @param {String|null} date stringified day in format 'YYYY-MM-DD'
+ * @return {Promise<void>}
+ */
+const syncBankAccounts = async (date = null ) => {
```

Comment:
Where `date` argument is used?
```suggestion
const syncBankAccounts = async (date = null ) => {
```

Reply:
It's not needed. I forgot to remove this after the tests for receiving bank account transactions